src/platform/forms/tests/save-in-progress/SaveInProgressIntro.unit.spec.jsx

```diff
@@ -508,13 +508,12 @@ describe('Schemaform <SaveInProgressIntro>', () => {
         prefillsAvailable: [],
       },
       login: {
-        currentlyLoggedIn: false,
+        currentlyLoggedIn: true,
```

Comment:
updating the component props so things are in a state so the custom header will render